app/components/channel_list/header/header.tsx

```diff
@@ -83,6 +88,13 @@ const ChannelListHeader = ({displayName, iconPad}: Props) => {
                     <CompassIcon
                         style={styles.plusIcon}
                         name={'plus'}
+                        onPress={async () => {
```

Comment:
This is temporal until the slideup panel is finished.

Reply:
@Willyfrog when are you planing to work on that ticket?

Reply:
Hopefully along this week I'll find some time to work on it, but if it is blocking, I'll happily let someone take it along with the branch with my changes. Just let me know if anyone wants/needs to work on it